src/Compiler/CMakeLists.txt

```diff
@@ -81,6 +81,7 @@ add_onnx_mlir_library(CompilerUtils
   LINK_LIBS PUBLIC
   ${OMLibs}
   CompilerPasses
+  Accelerator
```

Comment:
Do we need to add this? I see that you make InitAcelerator link to Accelerator.

Reply:
Reverted the change in InitAccelerator and avoid linking the Accelerator library because it already link the NNPAAccel.